lua/tardis/sh_menuoptions.lua

```diff
@@ -0,0 +1,42 @@
+if CLIENT then
+	hook.Add("PopulateToolMenu", "TARDIS2-PopulateToolMenu", function()
+		spawnmenu.AddToolMenuOption("Options", "Doctor Who", "TARDIS2_Options", "TARDIS Rewrite", "", "", function(panel)
+			panel:ClearControls()
+			-- Do menu things here
+			
+			local DLabel = vgui.Create( "DLabel" )
+			DLabel:SetText("The TARDIS Interface is available at any time using the button below:")
+			panel:AddItem(DLabel)
+			
+			local button = vgui.Create("DButton")
+			button:SetText("Open TARDIS Interface")
+			button.DoClick = function(self)
+				TARDIS:HUDScreen()
+			end
+			panel:AddItem(button)
+
+			local htoggle = vgui.Create("DCheckBoxLabel")
+			htoggle:SetText("Enable Health and Damage")
+			htoggle:SetValue(TARDIS:GetSetting("health-enabled"))
+			function htoggle:OnChange(val)
+				net.Start("TARDIS-SvCvars")
+					net.WriteString("tardis2_damage")
+					net.WriteType(val)
+				net.SendToServer()
+			end
+			panel:AddItem(htoggle)
+		end)
+	end)
+else
+	util.AddNetworkString("TARDIS-SvCvars")
+
+	net.Receive("TARDIS-SvCvars", function(len, ply)
```

Comment:
This allows arbitrary execution of commands, please limit it to just tardis2_damage and send up just the new value as an int don't send up the command too